.github/workflows/release.yml

```diff
@@ -3,7 +3,8 @@ name: Release SDK
 on:
   push:
     branches:
-      - 'main'
+      - '**' #just for testing, switch back to main after
```

Comment:
Is this all branches? I think we should specify one e.g. test and just push commits directly to it…

Reply:
It is all branches! I will close this PR right after testing and switch it back to main?